couchpotato/static/scripts/combined.base.min.js

```diff
@@ -187,7 +187,6 @@ var CouchPotato = new Class({
         }
```

Comment:
It is generated file, and all changes here are just consequence of changes in the `settings.js`